docs/content/tutorials/kic/deploy-with-kic.md

```diff
@@ -14,7 +14,11 @@ You can deploy NGINX Ingress Controller for Kubernetes with NGINX Service Mesh t
 
```

Comment:
I think it's worth mentioning somewhere what our mesh actually does to integrate ingress controller. It's important for users to know:
- adding a label to NIC that hasn't been created yet will result in NSM updating it when it's created
- adding a label to NIC that already exists requires a rollout restart for NSM to pick it up and update it

It's also probably worth mentioning which fields we add to the Pod, so a user knows what we're exactly changing.

Reply:
I added an initial section explaining what our webhook modifies. I think it gives a pretty detailed explanation, but let me know any changes you would like to see.